### PR TITLE
Fixes styling conflict caused by messaging page & adding icon to navbar

### PR DESCRIPTION
**Pull request by LeonZhang7**
- fixed css messaging conflicts and merging the messaging branch into main
- added messaging icon to image files
- added icon for messaging in navbar & routed it to messaging page
- **check branches "DirectMessaging" & "DM2" for individual commits**

TODO 
- [ ] fix actual styling of the messaging page

### CODE REVIEW

**Comment by LeonZhang7** [comment]
once merged, please delete branch

File: frontend/src/App.js
Comment (fatema-a) [inline frontend/src/App.js:1]: seems good just adding routing for messaging and is done the proper way

File: frontend/src/components/DirectMessage/Messaging.css
Comment (fatema-a) [inline frontend/src/components/DirectMessage/Messaging.css:1]: the addition of the css doesnt cause any problems

File: frontend/src/components/DirectMessage/Messaging.js
Comment (fatema-a) [inline frontend/src/components/DirectMessage/Messaging.js:1]: the addition of the actually file also does not cause any conflicts

File: frontend/src/components/Navbar.js
Comment (fatema-a) [inline frontend/src/components/Navbar.js:1]: the addition of it to the nav bar is done in the proper method as well

**Review by fatema-a — COMMENTED** [review]
all of these changes are good there are no conflicts and everything is done in the proper method

**Review by fatema-a — APPROVED** [review]

**Pull request merged**